src/token_store.h

```diff
@@ -46,6 +46,7 @@ class TokenStore {
   std::string override_client_id_;
   std::string override_client_secret_;
   std::string override_token_endpoint_;
+  std::string override_proxy_;
```

Comment:
You probably want to set this value in the same way as the other overrides, in `TokenStore::Read()`.

Reply:
I will add adjust TokenStore::Read() as well as TokenStore::Write()